src/java/org/apache/cassandra/tools/nodetool/AccordAdmin.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.tools.nodetool;
+
+import java.util.List;
+
+import io.airlift.airline.Arguments;
+import io.airlift.airline.Command;
+import org.apache.cassandra.tools.NodeProbe;
+import org.apache.cassandra.tools.NodeTool;
+
+public abstract class AccordAdmin extends NodeTool.NodeToolCmd
+{
+    @Command(name = "mark_stale", description = "Mark a replica as being stale and no longer able to participate in durability status coordination")
+    public static class MarkStaleCmd extends AccordAdmin
+    {
+        @Arguments(required = true, description = "One or more nodeIds to mark stale", usage = "<nodeId>+")
+        public List<String> nodeIds;
+
+        @Override
+        protected void execute(NodeProbe probe)
+        {
+            probe.getCMSOperationsProxy().accordMarkStale(nodeIds);
+        }
+    }
```

Comment:
TODO: We'll need a command to mark a node active again, but that should be a bit simpler...

Reply:
asked in slack... if a node is *alive* then should we allow it to be `stale`?  what about on reboot, do we take ourselves out of `stale`?

Reply:
Also, probably best to move this to an Accord-centered proxy...

Reply:
I think a case where we *do* want to remove a node from the "stale" set is when the node is removed from the `ClusterMetadata` `directory`, which is what populates the shards in an Accord `Topology` in the first place. It looks like this happens in `Transformer#replaced()` and `Transformer#unregister()`.

Reply:
There may also be some details we have to pay attention to around removing non-stale nodes when other members of a shard are stale. Removing without checking the staleness of other nodes in a shard might actually drop us below quorum, and that...would be bad.